src/app/modules/library/learning-resources/learning-resources.component.scss

```diff
@@ -0,0 +1,50 @@
+.c-filters {
+  display: flex;
+  justify-content: space-evenly;
+  background-color: white;
+}
+
+.c-card {
+  display: flex;
+  flex-wrap: wrap;
+  padding-top: 5%;
+}
+
+.example-header-image {
```

Comment:
give valid name and what is the background image about?

Reply:
Not needed removed